src/components/backend-ai-dialog.ts

```diff
@@ -160,14 +162,28 @@ export default class BackendAiDialog extends LitElement {
    * Open a dialog.
    */
   show() {
+    this.checkPreventEscapeKeyClose();
     this.dialog.show();
   }
 
```

Comment:
Why don't we just use string value (`close`) as default value which is the same way the `mwc-dialog` supports?